py/client/pydeephaven/updateby.py

```diff
@@ -57,7 +57,7 @@ class BadDataBehavior(Enum):
     """Allow the bad data to poison the result. This is only valid for use with NaN"""
 
 
-class OperationControl(UpdateByBase):
+class OperationControl(_UpdateByBase):
```

Comment:
Is sphinx ok with this, or should it just be public?

Reply:
Yeah, sphinx works great.